src/components/Button/button.scss

```
@@ -0,0 +1,21 @@
+.button {
+  height: 40px;
+  padding: 0 20px;
+  font-weight: bold;
+  background-repeat: no-repeat;
+  background-position: center;
+  color: black;
+  background-color: transparent;
+  cursor:pointer;
+
+  &--ghost{
+    padding: 0;
+    padding-left: 30px;
+    padding-right: 35px;
+    border: none;
+  }
+
+  &--blue{
+    color: hsl(234, 11%, 52%);
```

I recommend to create a scss variable to manage the colors, is it hard to know which color is this one 🤔 

Try to set on top of this file a variable like so:
```scss
$gray: hsl(234, 11%, 52%); // I have no idea what color is this haha
```

and then use it in the color property:
```suggestion
    color: $gray;
```